index.js

```diff
@@ -209,13 +193,12 @@ class MongoModels {
     }
 
 
-    static async findByIdAndUpdate() {
+    static async findByIdAndUpdate(...args) {
 
-        const args = argsFromArguments(arguments);
         const db = dbFromArgs(args);
         const collection = db.collection(this.collectionName);
         const id = args.shift();
-        const update = args.shift();
+        const update = { $set: args.shift() };
```

Comment:
This change sticks out at me and may be a breaking change for some consumers. What's the main motivation for this change?

Reply:
A change to mongodb now requires that an atomic be set for findOneAndUpdate (https://github.com/mongodb/node-mongodb-native/commit/eb68074). This appeared in version mongodb 3.1.0. This operator would be $set, $unset or $rename.

The intention here was to keep existing code working. Without the atomic operator the call will fail with "the update operation document must contain atomic operators."

I do think I need to handle this differently in case the arg passed in already has the operator.